Azure devops updates

ram-kishore-odi opened the pull request:
Can you please review the updates to the Azure DevOps training document ?

Thank you @britt-allen for the detailed review comments and for your time during the subsequent review meeting.
I sincerely appreciate your help.

I could not attach some of the images under the staging changes section (from your original document) as I do not have the original images. I plan to attach additional screenshots once dbtCloud and Azure DevOps integration is complete.

ian-r-rose commented on docs/git/devops.md:
```diff
@@ -1 +1,220 @@
+
+This guide provides a foundation for using Azure DevOps Repos and Azure DevOps for development projects. Refer to the following official Azure DevOps documentation for the additional information
+
+### **Writing Markdown**
```
@britt-allen I wonder if we should create a dedicated page for markdown?

britt-allen replied:
weird my comment didnt go through. i dont think we need to re-invent markdown docs. Ram links to some in here and I feel like it's already well-documented how to use markdown with the tools we choose.

ian-r-rose commented on docs/git/devops.md:
```diff
@@ -1 +1,220 @@
+* **Tables:** Use pipes (|) and hyphens (\-) to create tables.  
+  * *Example:* markdown | Header 1 | Header 2 | |---|---| | Row 1, Cell 1 | Row 1, Cell 2 | | Row 2, Cell 1 | Row 2, Cell 2 |
+
+**Key Differences and Considerations:**
```
I think we should specify that we are talking about differences with GitHub

britt-allen replied:
Agree

ram-kishore-odi replied:
Yes, I agree updated the title

ian-r-rose reviewed (commented):
This is looking great @ram-kishore-odi!

ram-kishore-odi commented on docs/git/devops.md:
```diff
@@ -1 +1,220 @@
+  * **Boards:** Track your work items and progress.  
+  * **Test Plans:** Manage your testing efforts.  
+* **Enhanced Security:** Azure DevOps provides robust access control and security features to protect your code.  
+* **Flexibility:** You can use your favorite Git client (like GitHub Desktop, the Git command line, or Visual Studio Code) to work with your code.
```
yes @ian-r-rose. Please see this documentation link - https://github.com/desktop/desktop/blob/development/docs/integrations/azure-devops.md

ian-r-rose replied:
Wow, good to know!

ram-kishore-odi commented on docs/git/devops.md:
```diff
@@ -1 +1,220 @@
+
+**Cloning the Project Repo Locally**
+
+[Git Credential Manager (GCM)](https://github.com/git-ecosystem/git-credential-manager), which makes the task of authenticating to Azure DevOps repositories easy and secure. GitHub Desktop now provides support for it. Installing git credential manager is recommended for working with Azure DevOps repos.
```
It does and using GitHub desktop may make things easier overall

ian-r-rose replied:
Agree, it has particularly been useful in cases where users' laptops are locked down enough that SSH ports are blocked. Carry on!

britt-allen commented on docs/git/devops.md:
```diff
@@ -1 +1,220 @@
+**Steps to Clone a repo**
+
+1. Go to your command line  
+2. Type cd \~.  
+3. Type git clone \<Azure DevOps repo URL\>.
+
```
yes cd works in both

ian-r-rose replied:
I'm referring to `~` as a shortcut for the home directory

britt-allen replied:
yes, from my google's i found that it does work. sorry i should have typed the whole command lol

ian-r-rose replied:
Sounds good, thanks

britt-allen commented on docs/git/devops.md:
```diff
@@ -1 +1,219 @@
+* As a reviewer, you can merge a PR after you approve it and all CI checks pass.  
+* Click the "Complete" button on the PR page to merge the changes.
+
+The basic writing and formatting syntax used on GitHub largely works in Azure DevOps as well. Both platforms utilize Markdown for formatting text in work items, pull requests, wikis, and other areas.
```
I don't think this info is needed. if you chose to keep it I think it makes sense to move it down to where you talk about markdown.

ram-kishore-odi replied:
Yes, I agree. I moved that sentence to the markdown section.

britt-allen commented on docs/git/devops.md:
```diff
@@ -1 +1,219 @@
+
+Writing markdown is important to learn when creating project documentation in markdown files (.md) and for writing Azure DevOps Work Items. This Microsoft [reference document](https://learn.microsoft.com/en-us/azure/devops/project/wiki/markdown-guidance?view=azure-devops) is a great starting point.
+
+**Common Markdown features supported in Azure DevOps:**
```
you could remove this section (lines 175 - 206) since what you link out to provides the exact same info

ram-kishore-odi replied:
Yes I removed it. Good catch !

britt-allen reviewed (dismissed):
I think after you implement Ian and I's remaining feedback and address your pre-commit failures this will be good to go. Thank you Ram!

ram-kishore-odi commented:
Thank you both for your great feedback ! I addressed the feedback items and also fixed pre-commit failures. Please let me know if you have any additional feedback. I will merge it into main tomorrow
